code/modules/cargo/expressconsole.dm

```diff
@@ -108,25 +158,28 @@
 				rank = "Silicon"
 			var/reason = ""
 			var/list/empty_turfs
-			var/area/landingzone
 			var/datum/supply_order/SO = new(pack, name, rank, ckey, reason)
 			if(!(obj_flags & EMAGGED))
 				if(SO.pack.cost <= SSshuttle.points)
 					landingzone = locate(/area/quartermaster/storage) in GLOB.sortedAreas
-					for(var/turf/open/floor/T in landingzone.contents)
+					for(var/turf/open/floor/T in landingzone.contents)//uses default landing zone
 						if(is_blocked_turf(T))
 							continue
 						LAZYADD(empty_turfs, T)
 						CHECK_TICK
 					if(empty_turfs && empty_turfs.len)
 						var/LZ = empty_turfs[rand(empty_turfs.len-1)]
+						if (istype(beacon) && usingBeacon)
+							LZ = get_turf(beacon)
+							beacon.update_status(SP_LAUNCH)
 						SSshuttle.points -= SO.pack.cost
+
 						new /obj/effect/DPtarget(LZ, SO, podID)
 						. = TRUE
 						update_icon()
 			else
 				if(SO.pack.cost * (0.72*MAX_EMAG_ROCKETS) <= SSshuttle.points) // bulk discount :^)

```

Comment:
The number on this line is different than the one on line 189, is this intentional?

Reply:
o shit